staging-app.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
The role of this file is not clear. Please add a comment about it's purpose and the expected syntax.